model/group_status.go

```diff
@@ -17,6 +17,12 @@ type GroupStatus struct {
 	InstallationsAwaitingUpdate int64
 }
 
+// GroupsStatus represents the status of a groups.
+type GroupsStatus struct {
+	ID     string
```

Comment:
We could probably just add the `ID` to `GroupStatus` and then just use `[]GroupStatus` for the return type, but this works. Just an idea for later maybe.